rllib/tests/test_rollout_worker.py

```diff
@@ -213,11 +213,10 @@ def test_callbacks(self):
         pg.train()
         pg.train()
         pg.train()
-        self.assertEqual(counts["sample"], 4)
+        self.assertGreater(counts["sample"], 0)
```

Comment:
This needed to change since we now always report metrics periodically based on min_iter_time_s, not timesteps.

Reply:
+1